packages/connect/src/api/ethereum/api/ethereumCall.ts

```diff
@@ -0,0 +1,65 @@
+// origin: https://github.com/trezor/connect/blob/develop/src/js/core/methods/EthereumCall.js
```

Comment:
this link does not exist

Reply:
removed, https://github.com/trezor/trezor-suite/pull/12122/commits/77ca47b832396734e096c825e9c11882c055deb4